src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/SchemaCompareIncludeExcludeNodeOperation.cs

```diff
@@ -66,27 +66,13 @@ public void Execute(TaskExecutionMode mode)
                     throw new InvalidOperationException(SR.SchemaCompareExcludeIncludeNodeNotFound);
                 }
 
-                // Check first if the dependencies will allow this if it's an exclude request
-                if (!this.Parameters.IncludeRequest)
-                {
-                    IEnumerable<SchemaDifference> dependencies = this.ComparisonResult.GetExcludeDependencies(node);
-
-                    bool block = dependencies.Any(d => d.Included);
-                    if (block)
-                    {
-                        this.Success = false;
-                        return;
-                    }
-                }
-
                 this.Success = this.Parameters.IncludeRequest ? this.ComparisonResult.Include(node) : this.ComparisonResult.Exclude(node);
 
-                // create list of affected dependencies of this request
-                if (this.Success)
-                {
-                    IEnumerable<SchemaDifference> dependencies = this.ComparisonResult.GetIncludeDependencies(node);
-                    this.ChangedDifferences = dependencies.Select(difference => SchemaCompareUtils.CreateDiffEntry(difference, null)).ToList();
-                }
+                // create list of dependencies of this schema difference
+                // if not successful, send exclude dependencies that caused it to fail
```

Comment:
Are we relying on the caller to interpret the `Dependencies` property correctly based on success?  If so, it'd probably be safer to set different properties depending out output, e.g. `this.AffectedDependencies` and `this.DependenciesToExclude`.

Reply:
True. I'll make them two separate properties